openapi/p4pa-organization.openapi.yaml

```
@@ -62,6 +62,128 @@ paths:
           description: Organization Not Found
         '500':
           description: Internal Server Error
+  /organization/{organizationId}/silService:
+    post:
+      tags:
+        - Organization
+        - SIL
+      summary: Save SIL service configuration for an organization with encryption
+      operationId: orgSilServiceCreate
+      parameters:
+        - name: organizationId
+          in: path
+          required: true
+          schema:
+            type: integer
+            format: int64
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/OrgSilServiceDTO'
+      responses:
+        '201':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/OrgSilServiceDTO'
+        '404':
+          description: Organization Not Found
+          content:
+            application/hal+json:
+              schema:
+                $ref: '#/components/schemas/OrgSilServiceErrorDTO'
+        '500':
+          description: Internal Server Error
+          content:
+            application/hal+json:
+              schema:
+                $ref: '#/components/schemas/OrgSilServiceErrorDTO'
+  /organization/{organizationId}/silService/{orgSilServiceId}:
+    get:
+      tags:
+        - Organization
+        - SIL
+      summary: Get SIL service configuration for an organization by ID decrypted
+      operationId: orgSilServiceGet
+      parameters:
+        - name: organizationId
+          in: path
+          required: true
+          schema:
+            type: integer
+            format: int64
+        - name: orgSilServiceId
+          in: path
+          required: true
+          schema:
+            type: integer
+            format: int64
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/OrgSilServiceDTO'
+        '404':
+          description: Organization Not Found
+          content:
+            application/hal+json:
+              schema:
+                $ref: '#/components/schemas/OrgSilServiceErrorDTO'
+        '500':
+          description: Internal Server Error
+          content:
+            application/hal+json:
+              schema:
+                $ref: '#/components/schemas/OrgSilServiceErrorDTO'
```

the error model is the same for all the APIs
```suggestion
                $ref: '#/components/schemas/OrganizationErrorDTO'
        '500':
          description: Internal Server Error
          content:
            application/hal+json:
              schema:
                $ref: '#/components/schemas/OrganizationErrorDTO'
```